modules/nodelist.php

```diff
@@ -58,7 +58,14 @@
 	$ng = $_GET['ng'];
 $SESSION->save('nlng', $ng);
 
-$nodelist = $LMS->GetNodeList($o, NULL, NULL, $n, $s, $g, $ng);
+$page = (!isset($_GET['page']) ? 1 : intval($_GET['page']));
+$per_page = intval(ConfigHelper::getConfig('phpui.nodelist_pagelimit', $listdata['total']));
+$offset = ($page - 1) * $per_page;
+$total = intval($LMS->GetNodeList($o, NULL, NULL, $n, $s, $g, $ng, null, null, true));
+
+$nodelist = $LMS->GetNodeList($o, NULL, NULL, $n, $s, $g, $ng, $per_page, $offset);
```

Comment:
to liczy rekordy spełniające warunek bez limitu 
może dam rade dzisiaj zrobić 
właśnie testuje coś w stylu "SELECT ?TOTAL? as total (..) "